src/docs/product/error-monitoring/states-triage/index.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Issue States and Triage
+sidebar_order:
+description: "Learn more about issue states and how to triage issues."
+---
+
+An issue can be in one of 3 states:
+
+- Unresolved: The default state for all new issues.
+- Ignored: The issue is ignored permanently or until a condition is met to move it to a state of Unignored. [Issue alerts](/product/alerts-notifications/issue-alerts/) are suppressed for ignored issues.
+- Resolved: The issue is resolved because it was fixed. You can also mark an issue as Resolved if you don't expect it to happen again.
```

Comment:
```suggestion
1. Unresolved: The default state for all new issues.
2. Ignored: The issue is ignored permanently or until a condition is met to move it to a state of Unignored. [Issue alerts](/product/alerts-notifications/issue-alerts/) are suppressed for ignored issues.
3. Resolved: The issue is resolved because it was fixed. You can also mark an issue as Resolved if you don't expect it to happen again.
```